### PR TITLE
Addressing Minor Cleanup

### PR DESCRIPTION
**Pull request by Anipik**
- Some feedback that got left in the initaial chunk
- Some merge conflicts were not resolved properly

### CODE REVIEW

File: src/libraries/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj [hunk 1]
```diff
@@ -4,13 +4,13 @@
...
     <IsPartialFacadeAssembly Condition="'$(TargetsNetFx)' == 'true'">true</IsPartialFacadeAssembly>
-    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetFrameworkSuffix)' == '' and '$(TargetsNetStandard)' == 'true'">SR.PlatformNotSupported_Registry</GeneratePlatformNotSupportedAssemblyMessage>
+    <GeneratePlatformNotSupportedAssemblyMessage Condition="'$(TargetsAnyOS)' == 'true' and '$(TargetsNetStandard)' == 'true'">SR.PlatformNotSupported_Registry</GeneratePlatformNotSupportedAssemblyMessage>
     <NoWarn Condition="'$(TargetsUnix)' == 'true'">$(NoWarn);CA1823</NoWarn> <!-- Avoid unused fields warnings in Unix build -->
     <TargetFrameworks>netstandard2.0-Windows_NT;netstandard2.0-Unix;netstandard2.0;net461-Windows_NT;$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix;$(NetFrameworkCurrent)-Windows_NT</TargetFrameworks>
     <ExcludeCurrentNetCoreApp>true</ExcludeCurrentNetCoreApp>
     <ExcludeCurrentFullFramework>true</ExcludeCurrentFullFramework>
   </PropertyGroup>
-  <ItemGroup Condition="'$(TargetsNetFx)' != 'true' and '$(TargetFrameworkSuffix)' != ''">
+  <ItemGroup Condition="'$(TargetsNetFx)' != 'true' and '$(TargetsAnyOS)' != 'true'">
```
Comment (safern) [inline src/libraries/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj]: I don't see where is `TargetsAnyOS` defined?

Comment (Anipik) [reply src/libraries/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj]: Its already merged in master https://github.com/dotnet/runtime/blob/master/src/libraries/Directory.Build.props#L116

Comment (safern) [reply src/libraries/Microsoft.Win32.Registry/src/Microsoft.Win32.Registry.csproj]: Oh I got confused by the other PR where we said we were going to define it 😄 -- cool, thanks.

**Review by safern — APPROVED** [review]

**Comment by ViktorHofer** [comment]
It's less than ideal that the merge conflicts weren't resolved in the original PR as history will look strange but I'm super happy that the config system change made it in.

**Comment by Anipik** [comment]
> It's less than ideal that the merge conflicts weren't resolved in the original PR 

They were kind of harmless deletions thats why when the ci went green, i merged them. :) but yes i agree that we should not do it.

**Review by ViktorHofer — APPROVED** [review]

**Comment by ViktorHofer** [comment]
Did some more cleanup and fixed the build failure.

**Comment by Anipik** [comment]
Thanks @ViktorHofer  for fixing the build.

File: src/libraries/System.Reflection.Emit.Lightweight/src/System.Reflection.Emit.Lightweight.csproj [hunk 2]
```diff
@@ -1,7 +1,8 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>$(NetCoreAppCurrent)-Windows_NT;$(NetCoreAppCurrent)-Unix</TargetFrameworks>
-    <Nullable>enable</Nullable>
```
Comment (safern) [inline src/libraries/System.Reflection.Emit.Lightweight/src/System.Reflection.Emit.Lightweight.csproj]: Why are you removing nullability?

Comment (safern) [reply src/libraries/System.Reflection.Emit.Lightweight/src/System.Reflection.Emit.Lightweight.csproj]: Oh nevermind, this doesn't have implementation anymore, right?

Comment (Anipik) [reply src/libraries/System.Reflection.Emit.Lightweight/src/System.Reflection.Emit.Lightweight.csproj]: yes that's the reason

**Review by safern — APPROVED** [review]

**Pull request merged**